src/main/java/org/jboss/pnc/bifrost/endpoint/LogUpload.java

```diff
@@ -93,6 +106,34 @@ public String uploadFinalLog(@Valid FinalLogUpload logUpload, @Context HttpHeade
         return "ok";
     }
 
+    @Path("/{processContext}/delete")
+    @DELETE
+    @RolesAllowed("**") // FIXME change to specific allowed roles
+    public Response deleteFinalLog(@PathParam("processContext") String processContext) {
```

Comment:
I think we need a `@Transactional` for the delete method.

Reply:
what a nice catch